### ISSUE TITLE
[data request] Deepfashion2

### ISSUE THREAD

**Issue by tasif498** [tensorflow/datasets#4785]
* Name of dataset: Deepfashion2
* URL of dataset: https://github.com/switchablenorms/DeepFashion2
* License of dataset: <license type>
* Short description of dataset and use case(s): DeepFashion2 is a comprehensive fashion dataset. It contains 491K diverse images of 13 popular clothing categories from both commercial shopping stores and consumers. It totally has 801K clothing clothing items, where each item in an image is labeled with scale, occlusion, zoom-in, viewpoint, category, style, bounding box, dense landmarks and per-pixel mask.There are also 873K Commercial-Consumer clothes pairs.
The dataset is split into a training set (391K images), a validation set (34k images), and a test set (67k images).

Folks who would also like to see this dataset in `tensorflow/datasets`, please thumbs-up so the developers can know which requests to prioritize.

And if you'd like to contribute the dataset (thank you!), see our [guide to adding a dataset](https://github.com/tensorflow/datasets/blob/master/docs/add_dataset.md).
